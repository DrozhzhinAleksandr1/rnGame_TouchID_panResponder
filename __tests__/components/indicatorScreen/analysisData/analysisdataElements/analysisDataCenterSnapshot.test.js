import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { AnalysisDataCenter } from '../../../../../app/components/indicatorScreen/analysisData/analysisDataElements/analysisDataCenter';

it('create snapshot AnalysisDataCenter', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<AnalysisDataCenter quotationText={'Pivot Points:'} quotationNum={'0.98139'} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
