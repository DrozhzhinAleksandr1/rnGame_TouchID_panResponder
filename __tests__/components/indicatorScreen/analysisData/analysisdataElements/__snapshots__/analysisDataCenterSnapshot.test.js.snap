// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot AnalysisDataCenter 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 2,
      "flexDirection": "row",
      "justifyContent": "center",
      "minHeight": 25,
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 14,
        "lineHeight": 16,
        "marginRight": 8,
        "marginTop": 10,
      }
    }
  >
    Pivot Points:
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Roboto-Light",
          "fontSize": 20,
        },
        Object {
          "paddingTop": 3,
        },
      ]
    }
  >
    0
    .
    9
    8
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Roboto-Light",
          "fontSize": 20,
        },
        Object {
          "fontSize": 23,
        },
      ]
    }
  >
    1
    3
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Roboto-Light",
          "fontSize": 20,
        },
        Object {
          "paddingTop": 3,
        },
      ]
    }
  >
    9
  </Text>
</Component>
`;
